content/md/first-steps/manage-your-attributes.md

```diff
@@ -289,8 +309,12 @@ Through the Delete shortcut in the grid:
 1. Click on the trash can icon
 1. Confirm the action in the popin
 
+::: warning
+You can delete attributes, deleting an attribute means that all information regarding this attribute will be removed from PIM. Once you have confirmed the action, the action cannot be reverted.
```

Comment:
This sentence is weird.

Reply:
changed into "Please note that deleting an attribute means that all information regarding this attribute will be removed from PIM. Once you have confirmed the action, it cannot be reverted."